### PR TITLE
Added new task, Clues

### PR DESCRIPTION
**Pull request by Beta-King**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]

### My steps to solve it,
[Mention the files changed. Add what changes you have done.] 

### Screenshots,
[If any]

### Links

http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

**Comment by Beta-King** [comment]
this exact PR has already experienced approvals in a different PR --> https://github.com/fossasia/labyrinth/pull/327 ; hence plz merge otherwise asking for reviews again :D

**Comment by samyakbvs** [comment]
Have you made an API for it?

**Comment by samyakbvs** [comment]
If not then pls do

**Comment by Beta-King** [comment]
API is not needed; people have to change one-line-code for each tile @samyakbvs

**Comment by samyakbvs** [comment]
@Beta-King If they just have to do one-line code changes then this PR is not worthy of becoming a task according to my point of view because in tasks there has to be substantial contributions

**Comment by samyakbvs** [comment]
Also from next time pls fill the PR template properly or delete it

**Comment by Beta-King** [comment]
one line code changes in different tiles; they have to make a trail of code changes in different tiles hence i think the contribution will be substantial :D

**Comment by Beta-King** [comment]
Ohkie, will remember to do that 👍

**Comment by Beta-King** [comment]
Besides as @niccokunzmann said, this could be a beginner task :D

**Comment by samyakbvs** [comment]
Okay agreed, I think this is good for a beginer's task

File: tasks/Clues.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth. In a labyrinth where people can lose their way, doomed to die in the maze; Clues and Riddles can be added to find your way and explore the various parts of the labyrinth can be a massive help and add an adventurous touch to the game !!
+
+Requirements:
+- In the `tiles.js` file edit the `alterInfo` tag of various tiles to lead the user to the various parts of the labyrinth (e.g. new year tile, etc)
+- The clues should lead to at-least a particular location(tile) in the Labyrinth
+- The clues should be interesting (e.g. riddles, etc)
+
+Expected Outcome:
+- A merged pull request
```
Comment (niccokunzmann) [inline tasks/Clues.md]: How many clues should be edited?

File: tasks/Clues.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Labyrinth: Add clues to lead to the various parts of the labyrinth !!
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth. In a labyrinth where people can lose their way, doomed to die in the maze; Clues and Riddles can be added to find your way and explore the various parts of the labyrinth can be a massive help and add an adventurous touch to the game !!
```
Comment (niccokunzmann) [inline tasks/Clues.md]: Yes, this is a clean motivation to me.

File: tasks/Clues.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Requirements:
+- In the `tiles.js` file edit the `alterInfo` tag of various tiles to lead the user to the various parts of the labyrinth (e.g. new year tile, etc)
```
Comment (niccokunzmann) [inline tasks/Clues.md]: This helps them, I think.

File: tasks/Clues.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Expected Outcome:
+- A merged pull request
+
+Hints:
+- If someone wants to work with you, let them do that. You can use techniques like [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q) or collaborate together on branches of your forks. You can attribute a pull-request also to other people who helped you out or submit a pull-request where you and an other person contributed together.
+- You can join our [chat on gitter](https://gitter.im/fossasia/labyrinth) if you get stuck to get help or just hang out and have a nice time with others.
+
+---
+
```
Comment (niccokunzmann) [inline tasks/Clues.md]: Here, you can add **Beginner**

**Comment by Beta-King** [comment]
Can we merge now ?? :D

**Comment by niccokunzmann** [comment]
This is great, I left more comments. 
From the start of GCI, I was asked to not add beginner tasks.
So, this one can be used in the next GCI or as a documentation for starters? What do you think?
Also, if this is for beginners, we could add a tutorial like try.github.io.

**Comment by Beta-King** [comment]
Yups seems good for next year, looking forward for this task in GCI2K18 :D

**Comment by Beta-King** [comment]
#362 --> solves this issue :D

**Pull request merged**